poutyne/framework/iterators.py

```diff
@@ -26,13 +27,23 @@ def _get_step_iterator(steps, generator):
 
 
 class StepIterator:
-    def __init__(self, generator, steps_per_epoch, batch_metrics_names, epoch_metrics_names, callback=None, mode=None):
+    def __init__(
+        self,
+        generator,
+        steps_per_epoch,
+        batch_metrics_names,
+        epoch_metrics_names,
+        torch_metrics_names,
+        callback=None,
+        mode=None,
+    ):
         # pylint: disable=too-many-arguments
         self.generator = generator
         self.steps_per_epoch = steps_per_epoch
         self.prefix = "" if mode == "train" else f"{mode}_"
         self.batch_metrics_names = [self.prefix + metric_name for metric_name in batch_metrics_names]
         self.epoch_metrics_names = [self.prefix + metric_name for metric_name in epoch_metrics_names]
+        self.torch_metrics_names = [self.prefix + metric_name for metric_name in torch_metrics_names]
```

Comment:
> It might become interesting to put this code chunk into a function like "format metric names".

Maybe if it becomes more complex but I think it's simple enough to keep that as is.

> Is there other forms of metrics we plan to add to poutyne?

Nope.